sdk/python/test_data/test_data_config.yaml

```diff
@@ -189,6 +189,9 @@ pipelines:
     - module: pythonic_artifacts_with_list_of_artifacts
       name: make_and_join_datasets
       execute: false
+    - module: cross_loop_after_topology
+      name: my_pipeline
+      execute: false
```

Comment:
This sample should be able to run on KFP OSS, right?
Otherwise, we aren't really "fixing" https://github.com/kubeflow/pipelines/issues/10050.

Reply:
Correct. Removed "fixes" instruction.

For posterity, the error we get when trying to submit to KFP is the following:

```txt
HTTP response body: {"error":"Failed to create a new run: InternalServerError: Failed to validate workflow for (): templates.entrypoint.tasks.root templates.root sorting failed: invalid dependency for-loop-2","code":13,"message":"Failed to create a new run: InternalServerError: Failed to validate workflow for (): templates.entrypoint.tasks.root templates.root sorting failed: invalid dependency for-loop-2","details":[{"@type":"type.googleapis.com/google.rpc.Status","code":13,"message":"In
```